var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -48,11 +51,22 @@ class Elpa(AutotoolsPackage, CudaPackage, ROCmPackage):
 
     variant("openmp", default=True, description="Activates OpenMP support")
     variant("mpi", default=True, description="Activates MPI support")
-    variant("gpu_streams", default=True, description="Activates GPU streams support")
+    variant(
+        "autotune",
+        default=False,
+        when="@2021.11.01",
```

Comment:
```suggestion
        when="@2021.11.01:",
```
According to the lines removed below.